flags.css

```diff
@@ -1,8 +1,56 @@
 /* Indonesia */
+.indonesiaFlag {
+    border: 2px solid;
+    height: 200px;
+    width: 300px;
+}
+.indonesiaStrip1 {
+    background-color: red;
+    height: 100px;
```

Comment:
I know it says to use pixels, but this will be easier as %

here are 3 ways of doing it that will suit you better

https://jsbin.com/mixapab/1/edit?html,css,output